tests/fixtures/components/item_activity.py

```diff
@@ -54,24 +59,75 @@ def generate(
         if container_type is ...:
             container_type = ContainerType.PROJECT
 
+        if item_id is ...:
+            item_id = str(self.fake.uuid4())
```

Comment:
`fake.uuid4()` returns a string representation of uuid by default. If you need UUID please use `cast_to=None` as param:
```suggestion
            item_id = self.fake.uuid4(cast_to=None)
```